### PR TITLE
Restore index page dropped content

### PR DESCRIPTION
**Pull request by mairaw**
While resolving some merge conflicts in one of my PRs, I've noticed that some content of the main page has been rolled back to an earlier version or content that we still wanted has been removed.

/cc @cartermp @GuardRex

### CODE REVIEW

**Comment by GuardRex** [comment]
@mairaw I only removed a Visual Studio 2015 ref from this. https://github.com/dotnet/docs/commit/50e128137fde445f64e10cf7c2a1ee5fdecb34e6#diff-d680e8a854a7cbad6d490c445cba2eba

I'm as innocent as newborn babe! 👶 ......... this time. :grinning:

**Comment by GuardRex** [comment]
oh ... you wanted that VS2015 ref back. I see. That was done on-purpose but apparently inadvertently.

File: index.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ title: .NET Documentation
                                                 </div>
                                             </div>
                                             <div class="cardText">
-                                                <h3><a href="/dotnet/articles/welcome">.NET Platform Guide</a></h3>
+                                                <h3><a href="/dotnet/articles/standard">.NET Platform Guide</a></h3>
```
Comment (GuardRex) [inline index.md]: Were we terminating the ones that will resolve to an *index.md* with a trailing slash, right?

*/dotnet/articles/standard/*

**Review by guardrex — CHANGES_REQUESTED** [review]

**Review by guardrex — APPROVED** [review]

**Comment by mairaw** [comment]
That one was added on purpose so people looking for the project.json docs could find it. 

I've addressed your comment.

**Comment by GuardRex** [comment]
Do you want to use the "verify-build-before-merge" every time an approval is made but before the build completes?

File: index.md [hunk 2]
```diff
@@ -120,11 +120,11 @@ title: .NET Documentation
...
                                             <div class="cardText">
-                                                <h3><a href="https://msdn.microsoft.com/library/w0x726c2(v=vs.110).aspx">.NET Framework Guide</a></h3>
-                                                <p><a href="https://msdn.microsoft.com/library/hh425099(v=vs.110).aspx">Get Started</a></p>
-                                                <p><a href="https://msdn.microsoft.com/library/hh156542(v=vs.110).aspx">Tutorials</a></p>
-                                                <p><a href="https://msdn.microsoft.com/library/54xbah2z(v=vs.110).aspx">Client Development</a></p>
-                                                <p><a href="https://msdn.microsoft.com/library/bb400852(v=vs.110).aspx">ASP.NET</a></p>
+                                                <h3><a href="/dotnet/articles/framework/">.NET Framework Guide</a></h3>
+                                                <p><a href="/dotnet/articles/framework/get-started/">Get Started</a></p>
+                                                <p><a href="/dotnet/articles/framework/development-guide">Tutorials</a></p>
```
Comment (GuardRex) [inline index.md]: One more ? ...

This one goes to ...

*.NET Framework Development Guide*

... not that that's wrong if that's where it should go, but it is a little bit :zap: shocking :zap: to land on something titled so differently and not obviously a list of tutorials.

**Comment by GuardRex** [comment]
@mairaw One more question :point_up: before it merges. I also raise this over at https://github.com/dotnet/docs/pull/2079/files#r117099449

File: index.md (hunk 2 above)
Comment (mairaw) [inline index.md]: I agree. Not sure how I missed that! I just simply replaced the URL.

**Comment by mairaw** [comment]
Good to go @GuardRex?

**Pull request merged**